### PR TITLE
[bc/zh-cn] Add Simplified Chinese Translation of `bc`

### PR DESCRIPTION
**Pull request by CloneWith**
- [x] I solemnly swear that this is all original content of which I am the original author
- [x] Pull request title is prepended with `[language/lang-code]` (example `[python/fr-fr]` or `[java/en]`)
- [x] Pull request touches only one file (or a set of logically related files with similar changes made)
- [x] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [x] If you've changed any part of the YAML Frontmatter, make sure it is formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [x] Yes, I have double-checked quotes and field names!

### CODE REVIEW

File: zh-cn/bc-cn.html.markdown [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        return 1 /*返回一个数值*/
+    }
+    return n * fac(n - 1) /*可以使用递归*/
+}
+
+/*不可使用闭包与匿名函数。*/
+
+num = fac(4) /*24*/
+
+/*这是本地变量的示例：*/
```
Comment (geoffliu) [inline zh-cn/bc-cn.html.markdown:77]: 局部变量

File: zh-cn/bc-cn.html.markdown [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+/*像 C 语言一样，只有 0 定义为假（false）。*/
+num = 0
+if(!num) {print "false\n"}
+
+/*与 C 语言不同，bc 没有 ?: 运算符。例如，
+ 这个代码块会导致出错：
+a = (num) ? 1 : 0
+但是你可以模拟一个：*/
+a = (num) && (1) || (0) /*&& 代表“和”，|| 代表“或”*/
```
Comment (geoffliu) [inline zh-cn/bc-cn.html.markdown:57]: s/和/与/

File: zh-cn/bc-cn.html.markdown [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+    print "Good morning\n" /*“print”输出字符串或变量，用英文逗号分隔。*/
+} else if(hour == 12) {
+    print "Hello\n"
+    /*字符串中的转义序列以反斜杠 \ 开头。
+    为了讲述清楚，这里是 bc 中
+    可用转义序列表的简化版：
+    \b: 退格
+    \c: 硬回车
+    \n: 新行
+    \t: 换挡符
```
Comment (geoffliu) [inline zh-cn/bc-cn.html.markdown:43]: 制表符

File: zh-cn/bc-cn.html.markdown [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+if(hour < 12) { /*运算符的用法与 C 语言类似。*/
+    print "Good morning\n" /*“print”输出字符串或变量，用英文逗号分隔。*/
+} else if(hour == 12) {
+    print "Hello\n"
+    /*字符串中的转义序列以反斜杠 \ 开头。
+    为了讲述清楚，这里是 bc 中
+    可用转义序列表的简化版：
+    \b: 退格
+    \c: 硬回车
+    \n: 新行
```
Comment (geoffliu) [inline zh-cn/bc-cn.html.markdown:42]: 换行符

File: zh-cn/bc-cn.html.markdown [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+lang: zh-cn
+---
+```bc
+/*这是一条
+多行注释。*/
+# 这（在 GNU bc 中）也是一条（单行）注释！
+
+    /*1. 变量与指令结构*/
+num = 45 /*所有变量都只以双精度浮点数形式存储，
+    并且你无法直接存储字符串常量。*/
```
Comment (geoffliu) [inline zh-cn/bc-cn.html.markdown:17]: 并且 bc 不支持直接存储字符串常量

File: zh-cn/bc-cn.html.markdown [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+num = 45; /*你可以选择在每个声明语句后
+    添加一个英文分号，也可以不加。*/
```
Comment (geoffliu) [inline zh-cn/bc-cn.html.markdown:19]: 每个语句后可以添加一个英文分号，也可以不加。

File: zh-cn/bc-cn.html.markdown [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+/*If 语句：*/
+hour = read() /*输入一个数字*/
+
+if(hour < 12) { /*运算符的用法与 C 语言类似。*/
+    print "Good morning\n" /*“print”输出字符串或变量，用英文逗号分隔。*/
+} else if(hour == 12) {
+    print "Hello\n"
+    /*字符串中的转义序列以反斜杠 \ 开头。
+    为了讲述清楚，这里是 bc 中
+    可用转义序列表的简化版：
```
Comment (geoffliu) [inline zh-cn/bc-cn.html.markdown:39]: 以下为 bc 中常用转义序列表

**Review by geoffliu — CHANGES_REQUESTED** [review]
@CloneWith 感谢！有几个地方的翻译是从英文直译，感觉有些生硬。我提了建议，麻烦看一下？

**Review by imba-tjd — APPROVED** [review]

**Review by geoffliu — APPROVED** [review]

**Pull request merged**